tools/sync_test_server/ros-testing-server.js

```diff
@@ -93,8 +96,6 @@ function stopRealmObjectServer(onSuccess, onError) {
         onSuccess();
     });
 
-    // Move back to `SIGTERM` once https://github.com/realm/ros/issues/234
-    // is resolved
```

Comment:
NO need to change back to `SIGTERM` ? Actually `SIGKILL` is better I think, it will surely kill the ROS.

Reply:
I was still having problems with `SIGTERM`. Not sure if SIGKILL is too heavy-handed, but it seems to get the job done.